lib/chromedriver.js

```diff
@@ -84,10 +86,7 @@ function getMostRecentChromedriver (mapping = CHROMEDRIVER_CHROME_MAPPING) {
   if (_.isEmpty(mapping)) {
     throw new Error('Unable to get most recent Chromedriver from empty mapping');
   }
-  return _.keys(mapping)
-    .map(semver.coerce)
-    .sort(semver.rcompare)
-    .map((v) => `${semver.major(v)}.${semver.minor(v)}`)[0];
+  return _.last(_.keys(mapping).sort(compareVersions));
```

Comment:
I thought semver is able to coerce such version numbers properly %/

Reply:
Semver only handles major, minor, and patch levels. Chromium (and now Chromedriver) uses four levels (not sure what their names are).

With the `semver` package `1.2.3.4` is the same as `1.2.3`.

Reply:
do we actually care about the last number there? This must be a build number. And I would not expect we have two different CD releases where only this number differs

Reply:
Currently they have releases [72.0.3626.69](https://chromedriver.storage.googleapis.com/index.html?path=72.0.3626.69/) and [72.0.3626.7](https://chromedriver.storage.googleapis.com/index.html?path=72.0.3626.7/), as well as [73.0.3683.20](https://chromedriver.storage.googleapis.com/index.html?path=73.0.3683.20/) and [73.0.3683.68](https://chromedriver.storage.googleapis.com/index.html?path=73.0.3683.68/).

So I'm not particularly willing to take too many assumptions.

Reply:
![fatality](https://www.mememaker.net/api/bucket?path=static/img/memes/full/2015/Nov/6/0/finish-him-fatality.jpg)